packages/webapp/src/views/common/legal-agreement-summary.html

```diff
@@ -46,6 +47,16 @@
         'classes': 'govuk-!-margin-bottom-7',
         'show':isLegalAgreementTypeS106Type
     },
+    {
+        'text': 'Other landowners or leaseholders on '+legalAgreementType,

```

Comment:
The legal agreement type needs converting to lower case.

![image](https://github.com/DEFRA/biodiversity-net-gain-service/assets/8915039/208c0258-7e0f-4a83-bd02-ff493dab639b)